no-kendoui-professional-in-package-manager.md

```diff
@@ -0,0 +1,67 @@
+
+---
+title: Kendo UI Professional Is Not Available in the AppBuilder Package Manager
+description: Kendo UI Professional Is Not Available in the AppBuilder Package Manager When Using a DevCraft License
+type: Troubleshooting
+page_title: Kendo UI Professional Is Not Available in the AppBuilder Package Manager
+slug: no-kendoui-professional-in-package-manager
+position: 0
+tags: devcraft, kendo, package manager, manage packages, appbuilder, license
+ticketid: 1017056, 1017115
+
+---
+
+## Environment 
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Package Manager</td>
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>
+  </tr>
+  <tr>
+    <td>Code (AppBuilder) client</td>
+    <td>All</td>
+  </tr>
+</table>
+## Description
+When having two subscriptions, the Devcraft subscription gives you the possibility to use the Kendo UI Professional. However, the Kendo UI Professional package cannot be installed in an AppBuilder app using the Package Manager.
+
+The Manage Packages window lets you install the Kendo UI Core package only. The Kendo UI Professional package is disabled, even though you have a valid Kendo UI Professional subscription.
+
+## Cause
+Currently, there is a limitation in the Package Manager that prevents it from recognizing a Kendo UI license from a bundle different than Telerik Platform. In Telerik Platform, only the Business and Enterprise subscriptions include a Kendo UI Professional license. So, if you have Telerik Platform Professional or Developer subscription in combination with a DevCraft subscription, the Package Manager won't let you install the Kendo UI Professional bundle.
```

Comment:
Currently, an AppBuilder limitation prevents the Package Manager from respecting your DevCraft subscription which provides access to Kendo UI Professional. The Package Manager respects only Telerik Platform subscription plans and is not aware of additional licenses for Telerik products. 

If you hold a Telerik Platform Professional or Developer subscription plan, you will not be able to install Kendo UI Professional from the Manage Packages wizard.